Description: Install header files to /usr/incude/ubertooth
 to make it easier to build firmware from source using
 the ubertooth-firmware-source package.
Author: Ruben Undheim <ruben.undheim@gmail.com>

Index: ubertooth/host/libubertooth/src/CMakeLists.txt
===================================================================
--- ubertooth.orig/host/libubertooth/src/CMakeLists.txt
+++ ubertooth/host/libubertooth/src/CMakeLists.txt
@@ -67,5 +67,5 @@ if( ${BUILD_STATIC_LIB} )
 endif( ${BUILD_STATIC_LIB} )
 
 install(FILES ${c_headers}
-		DESTINATION include
+		DESTINATION include/ubertooth
 		COMPONENT headers)
